templates/emails/collective.update.published.hbs

```diff
@@ -18,11 +18,19 @@ Subject: {{update.title}}
   <p><a href="{{url}}" style="font-size: 10pt;">view online</a></p>
 </center>
 
-<center>
-  <a href="{{config.host.website}}/{{collective.slug}}">
-    <img src="{{collective.previewImage}}" height="96" border=0 alt="{{collective.name}} logo" />
-  </a>
-</center>
+{{#if collective.previewImage}}
+  <center>
+    <a href="{{config.host.website}}/{{collective.slug}}">
+      <img src="{{collective.previewImage}}" height="96" border=0 alt="{{collective.name}} logo" />
+    </a>
+  </center>
+{{else}}
+  <center>
+    <a href="{{config.host.website}}/{{collective.slug}}">
+      <h3>{{collective.name}}</h3>
+    </a>
+  </center>
```

Comment:
Where you able to do a test rendering of it? Would be great to share a screenshot.

Reply:
Yeah I was